package.json

```diff
@@ -62,11 +58,28 @@
     "@types/tmi.js": "^1.8.2",
     "@types/twitch-ext": "^1.24.4",
     "editorconfig-checker": "^5.0.1",
-    "eslint": "^8.33.0",
+    "eslint": "^8.42.0",
+    "eslint-config-prettier": "^8.8.0",
     "eslint-config-react-app": "^7.0.1",
+    "eslint-plugin-prettier": "^4.2.1",
+    "husky": "^8.0.3",
+    "lint-staged": "^13.2.2",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.8.8",
     "react-app-rewire-multiple-entry": "^2.2.2",
     "react-app-rewired": "^2.2.1",
     "typescript": "^4.9.5",
     "webpack-image-resize-loader": "^5.0.0"
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx,json}": [
+      "eslint src --ext .ts,.tsx",
+      "prettier --check \"src/**/*.{js,jsx,ts,tsx,json}\""
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
```

Comment:
Is this needed? The husky docs seem to suggest you just need the pre-commit file?